lib/src/service/rule_service.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:async';
+
+import 'package:angular/angular.dart';
+import 'package:http/http.dart';
+
+import '../data_model/create_user_request.pb.dart';
+import '../data_model/rule.pb.dart';
+
+/// A class to interact with the Rules Builder server.
+@Injectable()
+class RuleService {
+  /// The base URL of the Rules Builder server.
+  static const serverUrl =
+      'https://rules-server-dot-spreadsheet-dv360-plugin.uc.r.appspot.com';
```

Comment:
it is better to put endpoint in the yaml file.

Reply:
On the client side, there isn't a yaml file or config file.